From 3c5044623c529dc991d1cf0e09fc9a1ad18696ad Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 10 Nov 2020 14:43:36 +0200
Subject: [PATCH 55/82] iot-gate-imx8: config: Update device config

1) CONFIG_FW_LOADER_USER_HELPER -- disable
2) CONFIG_PPP_FILTER -- enable
3) Remove redundant features.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/configs/iot-gate-imx8_defconfig | 18 ++++++++----------
 1 file changed, 8 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/configs/iot-gate-imx8_defconfig b/arch/arm64/configs/iot-gate-imx8_defconfig
index cd856e9c45d2..c6c91a436a76 100644
--- a/arch/arm64/configs/iot-gate-imx8_defconfig
+++ b/arch/arm64/configs/iot-gate-imx8_defconfig
@@ -157,8 +157,6 @@ CONFIG_PCIE_KIRIN=y
 CONFIG_PCIE_LAYERSCAPE_GEN4=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-CONFIG_FW_LOADER_USER_HELPER=y
-CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 CONFIG_BRCMSTB_GISB_ARB=y
 CONFIG_SIMPLE_PM_BUS=y
 CONFIG_VEXPRESS_CONFIG=y
@@ -263,7 +261,12 @@ CONFIG_MDIO_BUS_MUX_MMIOREG=y
 CONFIG_PPP=m
 CONFIG_PPP_BSDCOMP=m
 CONFIG_PPP_DEFLATE=m
+CONFIG_PPP_FILTER=y
+CONFIG_PPP_MPPE=m
+CONFIG_PPP_MULTILINK=y
+CONFIG_PPPOE=m
 CONFIG_PPP_ASYNC=m
+CONFIG_PPP_SYNC_TTY=m
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
@@ -393,16 +396,12 @@ CONFIG_REGULATOR=y
 CONFIG_REGULATOR_DEBUG=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_RC_CORE=m
-CONFIG_RC_DECODERS=y
-CONFIG_RC_DEVICES=y
+# CONFIG_RC_MAP is not set
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
-CONFIG_MEDIA_ANALOG_TV_SUPPORT=y
-CONFIG_MEDIA_DIGITAL_TV_SUPPORT=y
-# CONFIG_DVB_NET is not set
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=m
-CONFIG_VIDEO_OV5640=y
+# CONFIG_VIDEO_IR_I2C is not set
 CONFIG_IMX_DPU_CORE=y
 CONFIG_IMX_LCDIF_CORE=y
 CONFIG_DRM=y
@@ -507,7 +506,7 @@ CONFIG_DMADEVICES=y
 CONFIG_BCM_SBA_RAID=m
 CONFIG_FSL_EDMA=y
 CONFIG_FSL_EDMA_V3=y
-CONFIG_IMX_SDMA=y
+CONFIG_IMX_SDMA=m
 CONFIG_MV_XOR_V2=y
 CONFIG_MXS_DMA=y
 CONFIG_PL330_DMA=y
@@ -525,7 +524,6 @@ CONFIG_XEN_GNTDEV=y
 CONFIG_XEN_GRANT_DEV_ALLOC=y
 CONFIG_STAGING=y
 CONFIG_STAGING_MEDIA=y
-CONFIG_VIDEO_IMX_CAPTURE=y
 CONFIG_ION=y
 CONFIG_ION_SYSTEM_HEAP=y
 CONFIG_FSL_SDK_DPA=y
-- 
2.17.1

